src/main/java/gregtech/common/tileentities/machines/basic/GT_MetaTileEntity_Miner.java

```diff
@@ -0,0 +1,202 @@
+                            return;
+                        }
+                    }
+                    drillX++;
+                }
+            }
+        }
+    }
+
+    public boolean moveOneDown(IGregTechTileEntity aBaseMetaTileEntity) {
+        if (aBaseMetaTileEntity.getYCoord() + drillY - 1 < 0 || aBaseMetaTileEntity.getBlockOffset(0, drillY - 1, 0) == Blocks.bedrock) {
+            isPickingPipes = true;
+            return false;
+        }
+        if (aBaseMetaTileEntity.getBlockOffset(0, drillY, 0) == miningPipeTipBlock) {
+            aBaseMetaTileEntity.getWorld().setBlock(aBaseMetaTileEntity.getXCoord(), aBaseMetaTileEntity.getYCoord() + drillY, aBaseMetaTileEntity.getZCoord(), miningPipeBlock);
+        }
+        miningPipes:
+        {
+            for (int i = 0; i < mInputSlotCount; i++) {
+                ItemStack s = getInputAt(i);
+                if (s != null && s.getItem() == miningPipe.getItem() && s.stackSize > 0) {
+                    s.stackSize--;
+                    if (s.stackSize == 0) {
+                        mInventory[getInputSlot() + i] = null;
+                    }
+                    break miningPipes;
+                }
+            }
+            waitMiningPipe = true;
+            return false;
+        }
+        if (aBaseMetaTileEntity.getBlockOffset(0, drillY - 1, 0) != Blocks.air) {
+            mineBlock(aBaseMetaTileEntity, 0, drillY - 1, 0);
+        }
+        aBaseMetaTileEntity.getWorld().setBlock(aBaseMetaTileEntity.getXCoord(), aBaseMetaTileEntity.getYCoord() + drillY - 1, aBaseMetaTileEntity.getZCoord(), miningPipeTipBlock);
+        drillY--;
+        drillZ = -radius[mTier];
+        drillX = -radius[mTier];
+        return true;
+    }
+
+    public void mineBlock(IGregTechTileEntity aBaseMetaTileEntity, int x, int y, int z) {
+        ArrayList<ItemStack> drops = getBlockDrops(aBaseMetaTileEntity.getBlockOffset(x, y, z), aBaseMetaTileEntity.getXCoord() + x, aBaseMetaTileEntity.getYCoord() + y, aBaseMetaTileEntity.getZCoord() + z);
+        if (drops.size() > 0)
+            mOutputItems[0] = drops.get(0);
+        if (drops.size() > 1)
+            mOutputItems[1] = drops.get(1);
+        aBaseMetaTileEntity.getWorld().setBlockToAir(aBaseMetaTileEntity.getXCoord() + x, aBaseMetaTileEntity.getYCoord() + y, aBaseMetaTileEntity.getZCoord() + z);
+    }
+
+    private ArrayList<ItemStack> getBlockDrops(final Block oreBlock, int posX, int posY, int posZ) {
+        final int blockMeta = getBaseMetaTileEntity().getMetaID(posX, posY, posZ);
+        return oreBlock.getDrops(getBaseMetaTileEntity().getWorld(), posX, posY, posZ, blockMeta, 1);
+    }
+
+    @Override
+    public void saveNBTData(NBTTagCompound aNBT) {
+        super.saveNBTData(aNBT);
+        aNBT.setBoolean("isPickingPipe", isPickingPipes);
+        aNBT.setInteger("drillX", drillX);
+        aNBT.setInteger("drillY", drillY);
+        aNBT.setInteger("drillZ", drillZ);
```

Comment:
Nope, because after loading moveOneDown be called and do checking of input slots, not find pipes and set waitMiningPipe to true again.